examples/tutorial_1_connect.md

```diff
@@ -0,0 +1,108 @@
+Hi all, welcome to the first article of the nim-libp2p's tutorial series!
+
+_This tutorial is for everyone who is interested in building peer-to-peer chatting applications. No Nim programming experience is needed._
+
+To give you a quick overview, **Nim** is the programming language we are using and **nim-libp2p** is the Nim implementation of [libp2p](https://libp2p.io/), a modular library that enables the development of peer-to-peer network applications.
+
+Hope you'll find it helpful in your journey of learning. Happy coding! ;)
+
+# Before you start
+The only prerequisite here is [Nim](https://nim-lang.org/), the programming language with a Python-like syntax and a performance similar to C. Detailed information can be found [here](https://nim-lang.org/docs/tut1.html).
+
+Install Nim via their official website: [https://nim-lang.org/install.html](https://nim-lang.org/install.html)  
+Check Nim's installation via `nim --version` and its package manager Nimble via `nimble --version`.
+
+You can now install the latest version of `nim-libp2p`:
+```bash
+nimble install libp2p@#master
+```
+
+# A simple ping application
+We'll start by creating a simple application, which is starting two libp2p [switch](https://docs.libp2p.io/concepts/stream-multiplexing/#switch-swarm), and pinging each other using the [Ping](https://docs.libp2p.io/concepts/protocols/#ping) protocol.
+
+_You can extract the code from this tutorial by using `nim c -r tools/markdown_runner.nim examples/tutorial_1_connect.md`!_
```

Comment:
Maybe annotate with `TIP/HINT`? It took me a sec realizing that I was being hinted at something.